src/celeritas/ext/GeantPhysicsOptions.cc

```diff
@@ -35,10 +35,9 @@ char const* to_cstring(MscModelSelection value)
     static EnumStringMapper<MscModelSelection> const to_cstring_impl{
         "none",
         "urban",
-        "urban_extended",
-        "wentzel_vi",
-        "urban_wentzel",
-        "goudsmit_saunderson",
+        "wentzelvi",
+        "urban_wentzelvi",
+        "gs_wentzelvi",
```

Comment:
This may be dropped for now and added later when GS is actually implemented.

Reply:
True, I can drop GS for now. I may keep Wentzel VI in addition to Urban + Wentzel VI even though it may not be used alone in practice since it might be helpful in the (hopefully not too distant) future for validating that model separately from Urban.